modules/claims_api/app/controllers/claims_api/application_controller.rb

```diff
@@ -38,7 +38,7 @@ def header_request?
 
     def target_veteran(with_gender: false)
       if header_request?
-        headers_to_validate = %w[X-VA-SSN X-VA-First-Name X-VA-Last-Name X-VA-Birth-Date]
+        headers_to_validate = %w[X-VA-SSN X-VA-First-Name X-VA-Last-Name X-VA-Birth-Date X-Consumer-Username]
```

Comment:
we have x-va-consumer-username and x-consumer-username in a couple of places can you grep and unify with the correct one

Reply:
X-consumer-username is the correct one because it comes from Kong and is only suppose to be used for V0 where we need the consumer name as the source for the user stored in the models

Reply:
Yeah thought so as well can you change the others

Reply:
I'm not seeing an x-va-consumer-usernames anywhere, can you point to where in the code you're seeing it?

Reply:
Fixed